astro/src/content/docs/get-started/download-and-install/system-requirements.mdx

```diff
@@ -135,11 +135,27 @@ If you are running FusionAuth in an environment with no network access and are u
 
 If you are using the community plan, you can block all outbound network connectivity. Lack of connectivity will not affect the product. 
 
-Once on a paid plan, you must enable network access to FusionAuth, or your license will not function properly. Advanced features provided by FusionAuth Reactor cloud services require additional outbound network access.
+On a paid plan, you must enable network access to FusionAuth, or your features and license will not function properly.
```

Comment:
```suggestion
On a paid plan, you must enable network access to FusionAuth or the license and features will not function properly.
```

It is ok to leave as it, but I don't think the features are the users.  They are FusionAuth features.